src/test/java/spring/boot/com/sbparser/DataParseTest.java

```diff
@@ -0,0 +1,72 @@
+package spring.boot.com.sbparser;
+
+import org.apache.commons.csv.CSVRecord;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.List;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DataParseTest {
+    private static final String APPROPRIATE_DATA_FILE = "src/test/resources/test1.csv";
+    private static final String WRONG_FORMAT_DATA_FILE = "src/test/resources/test2.csv";
+    private static final String TEST_QUANTITY_DATA_FILE = "src/test/resources/test.csv";
+    private static final int REVIEWS_QUANTITY = 34;
+    private static final Review REVIEW_TO_COMPARE = new Review();
```

Comment:
Review is not an immutable class, so it can't be a constant
```suggestion
    private static final Review expectedReview = new Review();
```

Reply:
done